index.html

```diff
@@ -50,75 +50,16 @@ <h2 id="installation">
-            <th>Client</th>
-            <th>Installation instructions</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td><a href="https://github.com/ostera/tldr.jsx">Web client</a></td>
-            <td>Visit <a href="https://tldr.ostera.io">https://tldr.ostera.io</a></td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/tldr-pages/tldr-node-client">Node.js client</a></td>
-            <td><code>npm install -g tldr</code></td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/YellowApple/tldrb">Ruby client</a></td>
-            <td><code>gem install tldrb</code></td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/psibi/tldr-hs">Haskell client</a></td>
-            <td><code>stack install tldr</code></td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/skaji/perl-tldr">Perl client</a></td>
-            <td><code>cpanm App::tldr</code></td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/tldr-pages/tldr-python-client">Python client</a></td>
-            <td><code>pip install tldr</code> / <code>pacman -S tldr</code></td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/tldr-pages/tldr-c-client">C client</a></td>
-            <td><code>brew install tldr</code></td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/dbrgn/tealdeer/">Rust client</a></td>
-            <td><code>cargo install tealdeer</code> / <code>yay -S tealdeer</code></td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/freesuraj/TLDR">iOS client</a></td>
-            <td><a href="https://appsto.re/sg/IQ0-_.i">TLDR Man Page on App Store</a></td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/Moddus/tldr-python-dash-docset">Dash for macOS</a></td>
-            <td>
-              open <code>Preferences &gt; Downloads &gt; User Contributed</code> and find <code>tldr pages</code> in the list
-            </td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/pepa65/tldr-bash-client">Bash client</a></td>
-            <td><code>bpkg install pepa65/tldr</code></td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/isacikgoz/tldr">Go client</a></td>
-            <td><code>go get -u github.com/isacikgoz/tldr</code></td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/avih/tldr-fmt">Terminal-format-only POSIX shell script</a></td>
-            <td><code>git clone https://github.com/avih/tldr-fmt</code></td>
-          </tr>
-        </tbody>
-      </table>
       <p>
-        There are more clients listed in the
-        <a href="https://github.com/tldr-pages/tldr/wiki/tldr-clients">tldr clients wiki page</a>.
+        There are many more clients written in many languages that have sprouted from
```

Comment:
I don't think we need to emphasise the languages part.

```suggestion
        There are many more clients that have sprouted from
```